composer.json

```
@@ -30,7 +30,7 @@
     ],
     "require": {
         "php": ">=8.1",
-        "az-digital/arizona-bootstrap": "2.0.27",
+        "az-digital/arizona-bootstrap": "dev-33-update-composerjson-to-replace-v5-of-twbsbootstrap",
```

```suggestion
        "az-digital/arizona-bootstrap": "dev-main",
```
Change this once https://github.com/az-digital/arizona-bootstrap-packagist/issues/33 is merged